javascript/extraNetworks.js

```diff
@@ -345,7 +357,7 @@ async function refreshModelList({tabname}) {
     const model_type = currentTab;
     const currentPageTabsId = `${tabname}_${model_type}_current_page`;
     const currentPage = currentPageForTabs.get(currentPageTabsId);
-    fetchPageDataAndUpdateList({tabname, model_type, page: currentPage});
+    fetchPageDataAndUpdateList({tabname, model_type, page: currentPage, need_refresh: true});

```

Comment:
You might want to add refresh silently when user close the model lists without showing the blocking notification.